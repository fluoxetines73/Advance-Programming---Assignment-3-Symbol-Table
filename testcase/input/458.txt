INSERT b number
INSERT e_123_2r string
INSERT g_1 string
PRINT