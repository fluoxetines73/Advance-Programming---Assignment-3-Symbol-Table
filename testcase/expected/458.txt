success
success
success
b//0 e_123_2r//0 g_1//0